app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+
+class ProfileController extends Controller
+{
+    Public function index()
+    {
+        $user = Auth::user();
+
+        return view("auth.profile",
+        [
+            'user' => $user
+        ]);
+    }
+
+    public function edit(Request $request)
+    {
+        $user = Auth::user();
+
+        $validatedData = $request->validate([
+            'first_name' => 'required|string|max:255',
+            'last_name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users,email,' . $user->id,
+            'password' => 'sometimes|nullable|string|min:8|confirmed',
+            'password_confirmation' => 'required_with:password|same:password|min:8',
+            'postalcode' => 'required|string|max:10',
+            'street_name' => 'required|string|max:255',
+            'house_number' => 'required',
+            'phone_number' => 'required|string|max:15',
+            'experience' => 'required|string',
+            'comment' => 'nullable|string|max:255',
+        ]);
+
+        $user->first_name = $validatedData['first_name'];
```

Comment:
validatedData is een array met de dingen er in. Aangezien je de input names het zelfde hebt gemaakt als de veld namen zou je heel simpel $user->update($validatedData) kunnen doen